ddl/backfilling.go

```diff
@@ -56,8 +60,65 @@ const (
 	typeUpdateColumnWorker     backfillWorkerType = 1
 	typeCleanUpIndexWorker     backfillWorkerType = 2
 	typeAddIndexMergeTmpWorker backfillWorkerType = 3
+
+	// InstanceLease is the instance lease.
+	InstanceLease = 60 // s
 )
 
+// enableDistReorg means whether to enable dist reorg. The default is enable.
+// TODO: control the behavior
+var enableDistReorg = atomicutil.NewBool(false)
```

Comment:
We have feature tag in the ```sessionctx/variable/featuretag```. We can test feature when it enables and disables by controlling the feature tag. I think it may be useful for you.